Guide/Constructive/Bonetrousle/README.md

```diff
@@ -62,6 +32,12 @@ formula above so that it does work. Note that the problem says that any combinat
 the needed number of noodles is a valid solution.
 </p>
 
+<p>
+It is important to keep in mind that for this problem some variable values could exceed
+the value of an int in Java. Also, System.out.println(), may be too slow for big inputs.
```

Comment:
When using code syntax in markdown, format it like so: `System.out.println()` to make things easier